### ISSUE TITLE
Got this warning

### ISSUE THREAD

**Issue by adamkzh** [quanlieu/quan-node-playground#2]
**index.js:2178 Warning: Expected `onLoad` listener to be a function, instead got `false`.

If you used to conditionally omit it with onLoad={condition && value}, pass onLoad={condition ? value : undefined} instead.
    in img (created by t)
    in div (created by t)
    in div (created by t)
    in div (created by t)
    in t (created by t)
    in div (created by t)
    in div (created by t)
    in t (at Try.js:23)
    in div (at Try.js:22)
    in div (at Try.js:20)
    in Try (created by Route)
    in Route (at Routes.js:16)
    in div (at Routes.js:11)
    in Router (created by BrowserRouter)
    in BrowserRouter (at Routes.js:10)
    in Routes (at index.js:9)**

`<div> 
                <Header />
                <div className="images">
                    <SlideShow
                        images={this.imgUrl}
                        width="920px"
                        imagesWidth="800px"
                        imagesHeight="450px"
                        imagesHeightMobile="56vw"
                        thumbnailsWidth="920px"
                        thumbnailsHeight="12vw"
                        indicators thumbnails fixedImagesHeight
                    />
                </div>           
                <Footer />
            </div>`

 constructor(props){
        super(props);
        this.imgUrl = [
            'http://www/v/vspfiles/photos/897346002894-897346002993-5.png',
            'http://www./v/vspfiles/photos/897346002894-897346002993-8.png',
            'http://www./v/vspfiles/photos/897346002894-897346002993-4.png',
            'http://www./v/vspfiles/photos/897346002894-897346002993-2.png'
        ];
        
    }

**Comment by quanlieu** [comment]
images.js:224

`<img src={v} onLoad={i === 0 && this.handleLoaded} />`

Look like this is the cause, will fix right now. But it is just a warning, you can still get the result right?

**Comment by quanlieu** [comment]
BTW, please provide me with more information how to get the warning. In my develop environment it was silent.
Or you can make a PR.

**Comment by adamkzh** [comment]
yes, it's just a warning, I can get the result that I want. Here is the page.

```
import React, { Component } from 'react';
import Header from '../components/Header';
import SlideShow from 'react-image-show';
import Footer from '../components/Footer';

class Try extends Component{
    constructor(props){
        super(props);
        this.imgUrl = [
            'http://www./v/vspfiles/photos/897346002894-897346002993-5.png',
            'http://www/v/vspfiles/photos/897346002894-897346002993-8.png',
            'http://www./v/vspfiles/photos/897346002894-897346002993-4.png',
            'http://www.***/v/vspfiles/photos/897346002894-897346002993-2.png'
        ];
        
    }
   
    render(){
        return (
            <div> 
                <Header />
                <div className="images">
                    <SlideShow
                        images={this.imgUrl}
                        width="920px"
                        imagesWidth="800px"
                        imagesHeight="450px"
                        imagesHeightMobile="56vw"
                        thumbnailsWidth="920px"
                        thumbnailsHeight="12vw"
                        indicators thumbnails fixedImagesHeight
                    />
                </div>           
                <Footer />
            </div>
        )
    }
}

export default Try;
```

**Comment by quanlieu** [comment]
I don't know how to get the warning. Are you using some kind of linter? Change into 
`<img src={v} onLoad={i === 0 ? this.handleLoaded : undefined} />`
 is easy, but I don't want to bump a new version without making sure.

**Comment by quanlieu** [comment]
Never mind, I figured out. This is a warning come from create-react-app.
Fixed in `v0.2.4`
